tests/layer_tests/requirements.txt

```diff
@@ -12,9 +12,7 @@ transformers
 packaging
 pillow
 pytest
-networkx
```

Comment:
we could also remove networkx<=3.1.0 in tests/constraints.txt then

Reply:
it may be still needed for legacy MO tests